config/locales/en.yml

```diff
@@ -203,6 +203,15 @@ en:
         automatic_newsletter_frequency: "Send automatic newsletter: %{frequency_dropdown}"
         newsletter_daily: Daily
         newsletter_weekly: Weekly
+        delete_marketplace_title: "Delete marketplace"
+        type_marketplace_domain: "Type your marketplace domain (%{domain}) to the text field below:"
+        type_marketplace_domain_placeholder: "Type your marketplace domain here"
+        once_you_delete: "Once you delete the marketplace, you will not be able to access it anymore. Be careful."
+        are_you_sure: "Are you sure?"
+        i_understand_button: "I understand that by clicking this button my marketplace will be deleted"
+        last_community_updates: "Please note that after deletion, you may still receive the last marketplace updates email."
```

Comment:
Should this be:

Please note that after deletion, you and your customers may still receive the last marketplace update emails.